CondFormats/SiPixelTransient/src/SiPixelTemplate.cc

```diff
@@ -1056,6 +1057,12 @@ bool SiPixelTemplate::interpolate(int id, float cotalpha, float cotbeta, float l
       if(index_id_ < 0 || index_id_ >= (int)thePixelTemp_.size()) {
          throw cms::Exception("DataCorrupt") << "SiPixelTemplate::interpolate can't find needed template ID = " << id << std::endl;
       }
+
+      //check for nan's
+      if (!edm::isFinite(cotalpha) || !edm::isFinite(cotbeta)) {
+	success_ = false;
+	return success_;
+      }
```

Comment:
apparently due to a mix of tabs and spaces the indentation is not displaying appropriately in the web diff.
Please use spaces.

Reply:
this file in this release cycle has dos formatting. Is it fine if I just `dos2unix` it?

Reply:
I'd rather keep the line diffs in a backport to a minimum.